### DISCUSSION TITLE
cannot boot up on custom board

### DISCUSSION THREAD

**Discussion by billionfish** [zephyrproject-rtos/zephyr#50166] [Q&A]
**Describe the bug**
the samples/helloworld Demo can run and print well on BOARD_NRF52840DK_NRF52840. 

but this hex  file cannot boot up on  my board.

for simplest case, my custom board only use 4 pins (SWC SWD GND VCC) from [E73](https://www.aliexpress.com/wholesale?SearchText=E73%202G4M08S1C)

someone suggests set BOARD_ENABLE_DCDC=n  but not work for me.

so i need some help
thx

**Comment by IvanVnucec** [comment]
Can you get console output?

**Comment by billionfish** [comment]
> Can you get console output?

no output and other  information.

today I try to link the PIN to the LED but  it not lighted.

I guess the board even not boot..

**Comment by nordicjm** [comment]
Probably because you haven't connected a 32KHz crystal. This isn't a zephyr problem.

**Comment by billionfish** [comment]
> Probably because you haven't connected a 32KHz crystal. This isn't a zephyr problem.

[E73](https://www.aliexpress.com/wholesale?SearchText=E73%202G4M08S1C) has crystal.   And I can run other firmware(not zephyr based) on my board.
